src/openforms/forms/statistics.py

```diff
@@ -35,6 +35,11 @@ def export_registration_statistics(
     :arg limit_to_forms: A queryset of forms to limit the export to. If not provided or
       ``None`` is given, all forms are included.
     """
+    title_mappings = {
+        logevent.REGISTRATION_SUCCESS_EVENT: "Successfully registered",
+        logevent.FORM_SUBMIT_SUCCESS_EVENT: "Successfully completed",
+    }
```

Comment:
```suggestion
    title_mappings = {
        logevent.REGISTRATION_SUCCESS_EVENT: _("Successfully registered submissions between {start} and {end}"),
        logevent.FORM_SUBMIT_SUCCESS_EVENT: _("Successfully completed between {start} and {end}"),
    }
```

and then do `title=title_mappings[event].format(start=..., end=...)`